### ISSUE TITLE
Add a config file

### ISSUE THREAD

**Issue by kagarmoe** [DavidAnson/markdownlint-cli2-action#7]
I would like to pass a config file with the markdownlint-cli2-action. 
The config isn't at the root of the repo, but I can add it as a path or in an action.

I've tried this using the `download-file-action`, but I'm getting unexpected parsing errors on valid YAML.

```
      - uses: carlosperate/download-file-action@v1
        with:
          file-url: "https://github.com/chef/chef-web-docs/blob/main/.markdownlint.yaml"
          file-name: ".markdownlint.yaml"
          location: "./docs-chef-io"
      - name: Markdownlint
        uses: DavidAnson/markdownlint-cli2-action@v5
        with: 
          globs: "docs-chef-io/content/**/*.md" 
```

Error output:

```
Failed due to error: Error: Unable to parse '/home/runner/work/chef-workstation/chef-workstation/docs-chef-io/.markdownlint.yaml'; Parser 0: Unexpected token < in JSON at position 6; Parser 1: All collection items must start at the same column
```

**Comment by DavidAnson** [comment]
What you maybe really want us to invoke `markdownlint-cli2-config`, but thats not currently offered by the action. I'll add a TODO for myself. What seems like is happening with your current workflow is the YAML file isn't being saved right. Can you `cat` it after the download to check?

**Comment by DavidAnson** [comment]
I think you want to use the raw URL for the download:

https://github.com/chef/chef-web-docs/raw/main/.markdownlint.yaml

**Comment by DavidAnson** [comment]
(Well, that resolves to something direct, but I can't easily do so from my phone. Click it from a browser, then copy the URL out of the address bar.)

**Comment by kagarmoe** [comment]
Thanks! I'll try this

**Comment by koppor** [comment]
Allowing custom configs would allow to check [Keep-a-Changelog rules](https://github.com/avto-dev/markdown-lint/blob/master/lint/rules/changelog.js) for `CHANGELOG.md` files only.

**Comment by filiphagan** [comment]
@kagarmoe Not sure if this is what you're trying to achieve but my workaround is this:
I keep my config file in the config directory e.g. `/.github/workflows/config/.markdownlint.yml` and during CI pipeline run I copy this file to root before launching the linter. 

```
      # Markdown linting
      - name: Copy markdown config file to root
        run: |
          cp ./.github/workflows/config/.markdownlint.yml .markdownlint.yml
    
      - name: Lint all Markdown files
        uses: DavidAnson/markdownlint-cli2-action@v5
        with:
          globs: "**/*.md"
```

To integrate it with the VSCode plugin I pass this config path in `/.vscode/settings.json`

```
{
    "markdownlint.config": {
        "extends": "./.github/workflows/config/.markdownlint.yml"
    }
}
```
